scripts/AaveTokenScript.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Is this supposed to be imported somewhere? if not, and if objective is to be used to deploy the implementation, then it should be turned into a forge script